ts/Series/Column/ColumnSeries.ts

```diff
@@ -535,6 +535,9 @@ class ColumnSeries extends Series {
                 barX = plotX + seriesXOffset,
                 barW = seriesBarW;
 
+            point.negative = defined(point.y) &&
+                point.y < (series.options.threshold || 0);
```

Comment:
I think instead we should try to understand why `negative` was wrong in the first place. Consider this:

https://github.com/highcharts/highcharts/blob/9ff918be0104d77d0b1981e11e0daaa2c4c89ecd/ts/Core/Series/Series.ts#L2320-L2326

Judging by the comment, we added the `zoneAxis` stuff because we wanted bubbles to be negative when z was negative. As a side effect, it breaks when `zoneAxis` is `x`. It looks like we didn't think of that. There is no `options.xThreshold` property. So I think we should just rewrite the fix for #9728. Maybe check spesificly for `zoneAxis: 'z'`, or better still, move that part out to the Bubble series. It has a `translateBubble` function where we can set `negative`.